examples/wms-custom-tilegrid-512x256.js

```diff
@@ -25,7 +25,6 @@ var layers = [
     source: new ol.source.MapQuest({layer: 'sat'})
   }),
   new ol.layer.Tile({
-    extent: [-13884991, 2870341, -7455066, 6338219],
```

Comment:
I thought "extent" on the layer also prevented requesting tiles outside the extent. Do you know/understand the whole story? I'm not totally clear right now.